test/SparqlParserTest.cpp

```diff
@@ -1077,7 +1079,7 @@ TEST(ParserTest, testSolutionModifiers) {
                   .parse();
     ASSERT_EQ(1u, pq.children().size());
     ASSERT_EQ(1u, pq._orderBy.size());
-    EXPECT_THAT(pq, GroupByVariablesMatch({Variable{"?r"}}));
+    EXPECT_THAT(pq, m::GroupByVariablesMatch({Variable{"?r"}}));
```

Comment:
Should be `m::GroupByVariables`. The `is` or `match` part is already present via the namespace.